src/creationView/CreateView.js

```diff
@@ -0,0 +1,2316 @@
+import { Localizer, ActionHelper } from "../common/ActionSdkHelper";
+
```

Comment:
Too big file very difficult to understand for 3rd party app devs.

Please Breaks down into module chunks like :

util,uxutil, constant